test/test_excel_conversion.py

```diff
@@ -176,3 +177,66 @@ def test_roundtrip(filled_cldf_wordlist):
     }
 
     assert new_judgements == old_judgements
+
+
+def test_cell_comments():
```

Comment:
Gereon, does this Test hold already? The HyperLinkParser still contains "Cell_Comment". That's the only thing I'm not sure about. Looks good, besides that.